addons/binding/org.openhab.binding.mihome/ESH-INF/thing/aqara_86sw1.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mihome"
+                          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                          xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+                          xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="86sw1">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="bridge"/>
+        </supported-bridge-type-refs>
+
+        <label>Xiaomi Aqara 1 Channel Smart Light Control</label>
+        <description>Battery powered Aqara Switch with 1 Channel</description>
+
+        <channels>
+            <channel id="sw1" typeId="sw1"/>
+            <channel id="voltage" typeId="voltage" />
+            <channel id="batteryLevel" typeId="batteryLevel" />
+        </channels>
+
+        <config-description>
+            <parameter name="itemId" type="text" readOnly="true">
+                <label>Item ID</label>
+                <description>An identifier for the item connected.</description>
+                <required>true</required>
+            </parameter>
+        </config-description>
+    </thing-type>
+
+    <channel-type id="sw1">
+        <kind>trigger</kind>
+        <label>Button Event</label>
+        <event>
+            <options>
+                <option value="CLICK">click</option>
+                <option value="DOUBLE_CLICK">double click</option>
+            </options>
+        </event>
+    </channel-type>
+
+    <channel-type id="voltage" advanced="true">
```

Comment:
I see a lot of bindings providing an XML file called `channels.xml` then it is clear that they are shared.